Dockerfile

```diff
@@ -6,5 +6,5 @@ COPY environment.yml /
 COPY environment-percolator.yml /
 RUN conda env create -f /environment.yml && conda clean -a
 RUN conda env create -f /environment-percolator.yml && conda clean -a
-ENV PATH /opt/conda/envs/nf-core-mhcquant-1.1.0/bin:$PATH
+ENV PATH /opt/conda/envs/nf-core-mhcquant-1.2.0/bin:$PATH
 ENV PATH /opt/conda/envs/nf-core-mhcquant-percolator-1.0dev/bin:$PATH
```

Comment:
```suggestion
ENV PATH /opt/conda/envs/nf-core-mhcquant-percolator-1.2.0/bin:$PATH
```

Reply:
Will also require adding this to the respecitve environment.yaml, then we're good I guess 👍